bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -271,8 +271,8 @@ def setup_run_environment(self, env):
             export LD_LIBRARY_PATH=$libpath:\\$LD_LIBRARY_PATH"
 fi
 
-'{nrnivmodl}' -incflags '{incflags} '"$2" -loadflags \
-    '{loadflags} '"$extra_loadflags $3" "$1"
+'{nrnivmodl}' -incflags '{incflags} '"$3" -loadflags \
+    '{loadflags} '"$extra_loadflags $4" "$1"
```

Comment:
So what happens now if `$2` is not `--only-neuron`?

Reply:
You're right the way I had in mind was a bit cumbersome. Now it should be better and take care of all the flags